static/bidder-info/medianet.yaml

```diff
@@ -19,6 +19,9 @@ capabilities:
       - video
       - native
 userSync:
+  iframe:
+    url: https://hbx.media.net/checksync.php?cid=8CUEHS6F9&cs=87&type=mpbc&cv=37&vsSync=1&uspstring={{.USPrivacy}}&gdpr={{.GDPR}}&gdprsting={{.GDPRConsent}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}
+    userMacro: "<vsid>"
```

Comment:
![image](https://github.com/user-attachments/assets/a2a8e2f7-b0f9-477e-aba2-3c47728f0512)
Do You still support redirect?

Reply:
I still see support for redirect. I verified it still works.
<img width="1062" alt="Screenshot 2024-12-12 at 4 50 57 PM" src="https://github.com/user-attachments/assets/f148959b-a0d6-4fde-b555-986881bfefd9" />

Reply:
Yeah, medianet supports both. Thanks for quick verification.